frontend/public/locales/en/cluster.json

```diff
@@ -345,15 +346,15 @@
     "status.resuming.message": "The cluster is powering back on. Cluster operations and consumption of virtual machine and network resources resumes.",
     "status.stopping.message": "The cluster is powering down. Cluster operations and consumption of virtual machine and network resources will be suspended.",
     "status.degraded.message": "One or more of the cluster's add-ons are in a degraded state.",
-    "status.prehookjob.message": "The cluster pre-creation hooks are currently in progress.",
-    "status.prehookfailed.message": "One or more of the cluster pre-creation hooks failed.",
-    "status.posthookjob.message": "The cluster post-creation hooks are currently in progress.",
-    "status.posthookfailed.message": "One or more of the cluster post-creation hooks failed.",
+    "status.prehookjob.message": "The cluster prehook jobs are currently in progress.",
+    "status.prehookfailed.message": "One or more of the cluster prehook jobs failed.",
+    "status.posthookjob.message": "The cluster posthook jobs are currently in progress.",
+    "status.posthookfailed.message": "One or more of the cluster posthook jobs failed.",
     "status.importfailed.message": "The cluster failed to attach to the hub cluster.",
     "status.unknown.alert.title": "The cluster is not reachable",
     "status.offline.alert.title": "The cluster is not available",
-    "status.prehookfailed.alert.title": "The cluster pre-creation hooks failed",
-    "status.posthookfailed.alert.title": "The cluster post-creation hooks failed",
+    "status.prehookfailed.alert.title": "The cluster prehook failed",
+    "status.posthookfailed.alert.title": "The cluster posthook failed",
```

Comment:
since above it's `prehook jobs`, should this be `The cluster prehook jobs failed` ?